cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/CruiseControlRetriableConnectException.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.operator.resource.cruisecontrol;
+
+public class CruiseControlRetriableConnectException extends RuntimeException {
```

Comment:
I wonder if a better name for this would have been `CruiseControlRetriableConnectionException` to be explicit and avoid any possible confusion with Kafka Connect.

Reply:
Don't know ... I kind of like the relationship with the `ConnectException` we catch in the code to make this.

Reply:
I would agree with @tomncooper here. The `CruiseControlRetriableConnectException` seems weird to me as well. Double confusing with Kafka Connect being around.

Reply:
Also, it is thrown for `NoRouteToHostException` as well, not only for `ConnectException` which might add to another confusion.